strings/traumas.json

```diff
@@ -65,7 +65,9 @@
         "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?",
         "who the HELL do u thenk u r?!!!!",
         "without oxigen blob don't evoluate?",
-        "youed call her a toeugh bithc"
+        "youed call her a toeugh bithc",
+	"built DIFFERENT. I Stack Paper!",
+	"MAKE JIANT CRISTAL BURN I KEPT THROWING WELDERS IT WOOD LIGHT"
```

Comment:
```suggestion
	"MAKE JIANT CRISTAL BURN I KEPT THROWING WELDERS IT WOOD LIGHT",
```
Still missing the trailing comma, just makes expanding it easier later

Reply:
wait fuck its json i guess we dont do that there

Reply:
I already had this discussion with Tattle, it's a JSON list, not a BYOND one. Trailing comma errors out.

![image](https://github.com/tgstation/tgstation/assets/7019927/df20fae0-962b-495c-9467-216c2b6e777d)